.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+# Hitas Pull Request
+
+# Description
+
+&lt;write your pull request description here&gt;
+
+## Pull request checklist
+
+Check the boxes for each DoD items that has been completed:
+
+  - **Frontend**
+    - [ ] Changes have been tested
+  - **Backend**
+    - [ ] Changes have been tested
+    - [ ] Automatic tests has been added
+    - [ ] OpenAPI definitions have been updated
+    - [ ] Database migrations will work in test environment
```

Comment:
`[ ] Automatic tests has been added` already contains all of these? 🤔

Reply:
Tests do not check database migrations. For example it's possible to make a column required and that would fail when database contains data. We don't have any tests for loading a initial dataset, then applying a newest migration(s) and then checking migration will succeed